src/sentry/static/sentry/app/views/settings/components/forms/calendarWrapper.tsx

```diff
@@ -0,0 +1,4 @@
+import 'react-date-range/dist/styles.css';
+import 'react-date-range/dist/theme/default.css';
+
+export {Calendar as default} from 'react-date-range';
```

Comment:
should we maybe name this file `reactDateRangeCalendar` or `Calendar` or `CalendarField` ?